sdk/src/error.rs

```diff
@@ -224,6 +224,22 @@ pub enum Error {
     #[cfg(feature = "file_io")]
     OpenSslError(#[from] openssl::error::ErrorStack),
 
+    #[error(transparent)]
```

Comment:
Here I added as need arised as I was a bit unsure on what are the policies on c2pr error management.